pkg/module/utils.go

```diff
@@ -211,18 +211,30 @@ func subst(format string, jsonEncodedInputs ...string) string {
 // - inject the given values using template syntax
 // - JSON parse and check we don't have errors
 func LoadModule(module data.ModuleConfig, inputs map[string]string) (*data.Module, error) {
+	defer func() {
+		a := recover()
+		if a != nil {
+			log.Error().Any("panic", a).Msgf("panic handler")
```

Comment:
maybe make the message say "recovered panic in LoadModule" or is that obvious from the log?